pkg/cmd/pr/merge/merge.go

```diff
@@ -146,6 +152,7 @@ func NewCmdMerge(f *cmdutil.Factory, runF func(*MergeOptions) error) *cobra.Comm
 	cmd.Flags().BoolVarP(&opts.DeleteBranch, "delete-branch", "d", false, "Delete the local and remote branch after merge")
 	cmd.Flags().StringVarP(&opts.Body, "body", "b", "", "Body `text` for the merge commit")
 	cmd.Flags().StringVarP(&bodyFile, "body-file", "F", "", "Read body text from `file`")
+	cmd.Flags().StringVarP(&opts.Subject, "subject", "", "", "Subject `text` for the merge commit")
```

Comment:
Should this be `--title`?
Since other `gh` commands already use it I decided to go with `--subject` but I think `title` is more descriptive.

Reply:
I think `--subject` is fine